tests/items/profile_test.py

```diff
@@ -301,9 +301,9 @@ def test_virt_disk_driver(value, expected_exception):
 
 
 @pytest.mark.parametrize("value,expected_exception", [
-    ("", pytest.raises(ValueError)),
+    ("",  pytest.raises(ValueError)),
```

Comment:
There slipped an additional whitespace. Would be cool if you could remove that.

Reply:
Amende.